modules/best-practices/pages/gdpr-guidelines.adoc

```diff
@@ -0,0 +1,54 @@
+| Data of every runtime backed up every night. Additional backups are possible upon customer request.
+
+| Data retention
+| You manage how long you want to retain data. It is theoretically limitless, however the longer the data is retained, the more likely this will cause performance issue. Please refer to the “Process cleanup” section below to see how Bonita tools can help you purge your data.
+| Bonitasoft will store 15 days of back up for the production runtimes. This retention period can be increased upon customer request.
+
+| Restoration
+| Restorations are managed by you.Please note that the restoration procedure also depends on your database. 
+| Data restorations are done through a request and may result in scheduled maintenance. The service includes a restore to the latest backup. Bonitasoft is not responsible for the functional impacts of the rollback. 
+
+| Encryption
+| Encryption is managed by you. We recommended you use https://maven.apache.org/guides/mini/guide-encryption.html[Maven encryption]. Bonita also supports xref:setup-dev-environment:configure-maven.adoc[password encryption].
+| Bonita Cloud offers data in transit encryption with the latest HTTPS standard (TLS 1.3). Moreover, data at rest is also encrypted to secure your applications and processes data. Bonita Cloud uses the AES-256 encryption algorithm which is a recognized industry standard. 
+|===
+
+Please note that Bonitasoft has ISO 27001 certification for its Bonita Cloud information security and Bonita Cloud customer development, operations, and support.
+Enterprises who automate their business processes on Bonita Cloud want secure, robust enterprise applications with assurances that their data and privacy is safe. 
+Bonitasoft’s ISO 27001 certification is the result of a successful audit of the company’s cloud security practices by Bureau Veritas Certification, the world leader of certification.
+
+
+== Process cleanup
+
+[NOTE]
+====
+For Subscription editions only. +
+====
+
+To ensure compliance with data protection regulations or to facilitate general storage management, Bonita offers a specialized tool—referred to as the xref:runtime:purge-tool.adoc[Purge Tool]. This tool is designed to efficiently remove completed (archived) process instances from the Bonita Runtime environment.
+
+By default, Bonita Runtime retains all archives indefinitely. Given the operational mechanics of the Bonita Purge Tool, it is strongly recommended to execute this tool periodically, ideally during periods of minimal activity in the Studio. This can be automated through a crontab or an alternative scheduling method that best suits your needs.
```

Comment:
```suggestion
By default, Bonita Runtime retains all archives indefinitely. Given the operational mechanics of the Bonita Purge Tool, it is strongly recommended to execute this tool periodically, ideally during periods of minimal activity in the Runtime. This can be automated through a crontab or an alternative scheduling method that best suits your needs.
```